pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -94,8 +94,25 @@ def __init__(self, *args, **kwargs):
     def get_units(self, *filenames):
         """Gets the units to import and its total count."""
         units = []
+        all_filenames = set()
 
         for filename in filenames:
+            if not os.path.exists(filename):
+                continue
```

Comment:
Since this is outside the command `self.stdout.write()` can't be used, and raising a `CommandError` will stop the command immediately, so will try using just `print()`.